tmk_core/protocol/usb_descriptor_common.h

```diff
@@ -32,3 +32,23 @@
 #ifndef RAW_USAGE_ID
 #    define RAW_USAGE_ID 0x61
 #endif
+
+/////////////////////
+// Hires Scroll Defaults
+
+#ifdef POINTING_DEVICE_HIRES_SCROLL_ENABLE
+#    ifdef POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER
+#        if POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER > 127 || POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER < 1
+#            error "POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER must be between 1 and 127, inclusive!"
+#        endif
+#    else
+#        define POINTING_DEVICE_HIRES_SCROLL_MULTIPLIER 120
+#    endif
+#    ifdef POINTING_DEVICE_HIRES_SCROLL_EXPONENT
+#        if POINTING_DEVICE_HIRES_SCROLL_EXPONENT > 127 || POINTING_DEVICE_HIRES_SCROLL_EXPONENT < 0
+#            error "POINTING_DEVICE_HIRES_SCROLL_EXPONENT must be between 1 and 127, inclusive!"
+#        endif
+#    else
+#        define POINTING_DEVICE_HIRES_SCROLL_EXPONENT 0
```

Comment:
This isn't consistent.  Something here should be changed (either the text mentioning 1, or the 0's be changed to 1. 
```suggestion
#        if POINTING_DEVICE_HIRES_SCROLL_EXPONENT > 127 || POINTING_DEVICE_HIRES_SCROLL_EXPONENT < 0
#            error "POINTING_DEVICE_HIRES_SCROLL_EXPONENT must be between 1 and 127, inclusive!"
#        endif
#    else
#        define POINTING_DEVICE_HIRES_SCROLL_EXPONENT 0
```

Reply:
Thanks for catching this. It is the error message and documentation that is incorrect. I have pushed a correction.